x-pack/solutions/security/packages/features/src/security/product_feature_config.ts

```diff
@@ -32,25 +59,67 @@ export const securityDefaultProductFeaturesConfig: DefaultSecurityProductFeature
       },
     },
   },
-  [ProductFeatureSecurityKey.investigationGuide]: {
+
+  [ProductFeatureSecurityKey.externalDetections]: {
     privileges: {
       all: {
-        ui: ['investigation-guide'],
+        ui: ['external_detections'],
+        api: [],
       },
       read: {
-        ui: ['investigation-guide'],
+        ui: ['external_detections'],
+        api: [],
+      },
+    },
+  },
+  [ProductFeatureSecurityKey.detections]: {
+    management: {
+      insightsAndAlerting: ['triggersActions'],
+    },
+    alerting: alertingFeatures,
```

Comment:
Hey @tomsonpl, I'm pretty sure that without the proper alerting privileges configured, users won’t be able to install or run any detection rules, including the promotion rules required for AI4SOC.  

Can you check if rules are installable and runnable under the AI4SOC tier?

Reply:
@xcrzx good catch, thanks! my assumption was as we discussed that the rules are not available :) 
I reverted the changes, now it should work as expected  👍